### PR TITLE
Added dynamic generation of a manifest to make use of theming capabilities in modern browsers.

### PR DESCRIPTION
**Pull request by LeoKlaus**
This sets the correct values for "theme_color" and "background_color" according to the users preferences in a dynamically generated manifest.

With this change, the address bar on most modern mobile browsers (and Safari on Mac) adapts to the color of the user interface.

If you can provide a higher resolution version of the favicon, I'll happily add the corresponding icons to further improve the appearance of the PWA.

### CODE REVIEW

File: packages/client/src/App.vue [hunk 1]
```diff
@@ -56,6 +58,49 @@ export default {
...
+      background_color : pwaBackground,
+      display : 'standalone',
+      scope : '/',
+      start_url : '/#/scan',
+      name : 'scanservjs',
+      short_name : 'scanservjs',
+      description : 'SANE scanner nodejs web ui'
+    };
+    let content = encodeURIComponent(JSON.stringify(manifest));
+    let url = 'data:manifest+json,' + content;
```
Comment (sbs20) [inline packages/client/src/App.vue:98]: Inline and use backtick intepolation:

```javascript
const url = `data:manifest+json,${encodeURIComponent(JSON.stringify(manifest))}`;
```

**Review by sbs20 — COMMENTED** [review]
Nice. Thank you for this. And sorry for a ridiculously slow reply.

I have added new icons here: https://github.com/sbs20/scanservjs/pull/459

Feel free to move them.

**Comment by sbs20** [comment]
Let me know when you've finished and I'll take a look through.

(Ignore the build failure)

**Comment by LeoKlaus** [comment]
Should be done now. I don't have an Android device to test the icons. They don't appear to work on iOS, though I suspect that's related to my instance not using HTTPS.

**Pull request merged**